### PR TITLE
DOCSP-42340 Multikey Index

### PR DESCRIPTION
**Pull request by lindseymoore**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-scala/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-42340>
Staging - <https://deploy-preview-76--docs-scala.netlify.app/indexes/multikey-index/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/indexes/multikey-index.txt [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+on the ``cast`` field:
+
+.. literalinclude:: /includes/indexes/multikey-field.scala
+   :start-after: start-index-multikey
+   :end-before: end-index-multikey
+   :language: scala
+   :copyable:
+   :dedent:
+
+You can verify that the index was created by using the ``listIndexes()`` method.
```
Comment (jordan-smith721) [inline source/indexes/multikey-index.txt:75]:
```suggestion
You can verify that the index was created by calling the ``listIndexes()`` method.
```

File: source/indexes/multikey-index.txt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+   .. output::
+      :visible: false
+
+      {"_id":...,"title":"Fanaa",...,"cast": ["Aamir Khan", "Kajol", "Rishi Kapoor", "Tabu"],...}
+
+Additional Information
+----------------------
+
+To learn more about compound indexes, see :manual:`Multikey
```
Comment (jordan-smith721) [inline source/indexes/multikey-index.txt:112]:
```suggestion
To learn more about multikey indexes, see :manual:`Multikey
```

**Review by jordan-smith721 — APPROVED** [review]
One suggestion + one typo fix but otherwise LGTM!

File: source/includes/indexes/multikey-field.scala [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import com.mongodb.{ServerApi, ServerApiVersion}
+import org.mongodb.scala.{ConnectionString, MongoClient, MongoClientSettings}
+import org.mongodb.scala.bson.Document
+import Helpers._
+
+// start-multikey-index-imports
+import org.mongodb.scala._
+import org.mongodb.scala.model.Indexes
+import org.mongodb.scala.model.IndexOptions._
+import org.mongodb.scala.model.Projections._
```
Comment (jyemin) [inline source/includes/indexes/multikey-field.scala:12]: Lots of these imports are unused

**Review by jyemin — CHANGES_REQUESTED** [review]
Please, please, please compile and run all the code before putting it up for review.  I got lucky spotting this error (I'm not a very good compiler!).

File: source/includes/indexes/multikey-field.scala (hunk 3 above)
Comment (lindseymoore) [reply source/includes/indexes/multikey-field.scala]: Thanks! Removed projections and sorts

File: source/includes/indexes/multikey-field.scala [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    val database: MongoDatabase = mongoClient.getDatabase("sample_mflix")
+    val collection: MongoCollection[Document] = database.getCollection("movies")
+    // end-db-coll
+
+    // start-index-multikey
+    collection.createIndex(Indexes.ascending("cast")).printResults()
+    // end-index-multikey
+
+    // start-index-multikey-query
+    val filter1 = and(equal("cast", "Aamir Khan"), equal("cast", "Kajol"))
```
[the remark this answers is not in the document]
Comment (lindseymoore) [reply source/includes/indexes/multikey-field.scala]: Thank you, fixed!

**Review by mcmorisi — APPROVED** [review]
LGTM!

**Pull request merged**